File: tmdb-core/src/main/java/dev/reuss/tmdb/domain/configuration/ConfigurationPaths.java
package dev.reuss.tmdb.domain.configuration;

final class ConfigurationPaths {

    private static final String CONFIGURATION = "/configuration";

    private ConfigurationPaths() {
    }

    static String details() {
        return CONFIGURATION;
    }

    static String countries() {
        return CONFIGURATION + "/countries";
    }

    static String jobs() {
        return CONFIGURATION + "/jobs";
    }

    static String languages() {
        return CONFIGURATION + "/languages";
    }

    static String primaryTranslations() {
        return CONFIGURATION + "/primary_translations";
    }

    static String timezones() {
        return CONFIGURATION + "/timezones";
    }
}

File: tmdb-core/src/main/kotlin/dev/reuss/tmdb/domain/configuration/ConfigurationPaths.kt
package dev.reuss.tmdb.domain.configuration

import dev.reuss.tmdb.core.path.tmdbPath

internal object ConfigurationPaths {
    fun details(): String = tmdbPath("configuration")

    fun countries(): String = tmdbPath("configuration", "countries")

    fun jobs(): String = tmdbPath("configuration", "jobs")

    fun languages(): String = tmdbPath("configuration", "languages")

    fun primaryTranslations(): String = tmdbPath("configuration", "primary_translations")

    fun timezones(): String = tmdbPath("configuration", "timezones")
}
